### DISCUSSION TITLE
Looking for python command to return root layer of stage

### DISCUSSION THREAD

**Discussion by thomashollier** [Autodesk/maya-usd#1470] [Q&A]
I need to build a full scene procedurally and would like to access the root layer created by the "mayaUsdProxyShape" node so I can create sublayers from it.

**Comment by kxl-adsk** [comment]
Hi @thomashollier. Your question should be answered by this discussion - https://github.com/Autodesk/maya-usd/discussions/1264

BTW. I will convert this issue to discussion forums. Please in the future use GitHub discussion to connect with us.
